crates/bevy_app/src/plugin_group.rs

```diff
@@ -96,6 +96,21 @@ impl PluginGroupBuilder {
         self
     }
 
+    pub fn remove<T: Plugin>(&mut self) -> &mut Self {
```

Comment:
I like this change; can you spin it out into a seperate tiny PR?

Reply:
Here it is: https://github.com/bevyengine/bevy/pull/2171

Reply:
Based on discussion at pull request, this will be migrated after #2039 has been merged